### ISSUE TITLE
Visualise in the detail view that certain information is present

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#333]
It would be useful if we could visualise somehow in the panel headers that certain information is present, so that users can open individual panels in a more useful way. Suggestion: by a right-aligned greyscale icon. The most urgent one would be for Notes: if there are one or more notes, display a 'blank document' icon at the right of the panel header -- so that the user doesn't click on it hoping to find something but finding nothing.
Further extensions:
* Other Media: display a movie or picture icon depending on whats in it;
* Relations to Other/Foreign Signs: display a double arrow icon if at least one relation has been defined;
* Frequency: perhaps one to three stars to indicate the sign occurs only once, between 1 and 5 times, and 6 times or more.
* Phonology: display an icon (suggestions welcome) for all signs for which both Handedness and Handshape Strong are filled.

Not needed for:
* Morphology: less urgent: any morphemes are already displayed alongside the gloss; 
* Semantics: info too heterogeneous
* Language & Dialect: this will need some revision in the future anyway.

**Comment by susanodd** [comment]
Okay, I nearly finished this. Frequency is left.
One question: for the Other Media, if you look at what kinds of things you can choose, it doesn't actually look like photos or videos. Are some of these data files?
At the moment, I am using a camera icon.

**Comment by ocrasborn** [comment]
No, other media are generally videos in our case, but could be images as well. No other type of data foreseen.

**Comment by susanodd** [comment]
![pulldown-menu-other-media](https://user-images.githubusercontent.com/24272544/29355197-5e56f98e-8270-11e7-9d10-554c020a6208.png)

**Comment by susanodd** [comment]
In the interface, it's not clear (to me) what those things are. None of them look like media?

**Comment by susanodd** [comment]
Phonology, out of curiousity, the Handedness, 1, 2a, 2s, X. What do those mean?
I found various icons with hands and i'm looking to see if more info can be embeded.
![raised_hand_with_fingers_splayed](https://user-images.githubusercontent.com/24272544/29356511-cabdc0b8-8274-11e7-8f22-0497b0d867d8.png)
![reversed_raised_hand_with_fingers_splayed](https://user-images.githubusercontent.com/24272544/29356515-cf730348-8274-11e7-8f4f-f177bfef5fdc.png)
![waving_hand_sign](https://user-images.githubusercontent.com/24272544/29356522-d3cb2b96-8274-11e7-8e8d-acf09f29bcdb.png)
![clapping_hands_sign](https://user-images.githubusercontent.com/24272544/29356535-dde414b2-8274-11e7-9dab-cac7aa07f9b1.png)
![open_hands_sign](https://user-images.githubusercontent.com/24272544/29356532-dbb410ac-8274-11e7-9c9c-427d9dc98728.png)
![happy_person_raising_one_hand](https://user-images.githubusercontent.com/24272544/29356536-de3cba68-8274-11e7-8825-f69ae60661d2.png)
![turned_ok_hand_sign](https://user-images.githubusercontent.com/24272544/29356557-eb68b714-8274-11e7-8fd8-8d3248e7d42f.png)

**Comment by susanodd** [comment]
Okay, I pushed a version with icons for all the above data. (On master branch github.)
I added a File Type column to the Other media section.
(The example below has fake data added to it on my local copy.)

![icons-detail-view](https://user-images.githubusercontent.com/24272544/29361004-d989e548-8285-11e7-9b14-67c4b4fe5de1.png)

**Comment by ocrasborn** [comment]
Nice @susanodd ! Answers to your questions:
* The 'Type' of other media refers to the project they stem from, but they will always be representations (in image or video) of the selected sign.
* The classification of handedness refers to whether a sign is one-handed (1), two-handed symmetrical (s), two-handed but asymmetrical (2a). The 'X' is a shortcut for saying 'this gloss is abstract or variable, and so we cannot enter any phonological properties including handedness. Not the most elegant solution perhaps, but as this field is always used for any sign, the X provides a nice and quick way of searching these special glosses. Anyway, I like the image you selected now, the single hand with all fingers extended, it's most abstract.

**Comment by ocrasborn** [comment]
I changed my mind about the semantics panel (sorry :-)): could you add an icon there if a value has been selected for the field 'Semantic field'? (Reason: we should strive to always fill that field, and to add further values if the present ones don't suffice, so it's useful to see when it has not yet been filled.)
As to the icon here... A simple black-filled circle of the size of the stars? Any better suggestion welcome!

**Comment by susanodd** [comment]
A black circle icon has been added for the Semantics panel.
(Pushed to github master branch.)

**Comment by ocrasborn** [comment]
Hi @susanodd , could you check out this sign: https://signbank.science.ru.nl/dictionary/gloss/4362/ The symbol says 'relations specified', but I see nothing.
Another issue: the one-star frequency symbol is displayed much too large (sometimes? in my mac/safari only?).

**Comment by susanodd** [comment]
Okay, this has been fixed. Pushed to master branch on github.

**Comment by susanodd** [comment]
I added a hyperlink so that when the gloss's Lemma Group has more than one gloss in it, the hyperlink takes you to the Lemma View List for that Lemma Group.
I implemented this simply as browser link with parameters to the search page, with ^ and $ around the Lemma name.
There is no hyperlink shown if there are no other glosses in the same Lemma Group.

![gloss-detail-sign-with-lemma-group-link](https://user-images.githubusercontent.com/24272544/32107443-e15fbffe-bb2e-11e7-8667-f031cd30618a.png)
![result-of-lemma-group-link](https://user-images.githubusercontent.com/24272544/32107451-e6d68d6e-bb2e-11e7-9eb1-5e51ec1a77e5.png)
